src/pages/PlayerList.js

```diff
@@ -1,20 +1,44 @@
 import React from 'react';
+import { alpha } from '@mui/material/styles';
+import {Box, Table, TableBody, TableCell, TableContainer, TableHead, TableRow, TableSortLabel, Toolbar, IconButton, Tooltip, Paper, Checkbox} from '@mui/material';
+import DeleteIcon from '@mui/icons-material/Delete';
 
 const apiUrl = process.env.REACT_APP_SERVER_URL
 
-function PlayerList (props) { 
+function PlayerList (props) {
     const players = props.playersList
+    const selectedPlayers = players.filter((player) => player.selected === true);
+    const IsSelected = selectedPlayers.length > 0;
+
+    const headCells = [
```

Comment:
This kind of constant value is better to be declared outside of the function(component). Because this data wouldn't change. And the array creation inside of the function(component) would be repeated every time when the react component re-renders, which is not necessary.